Tests/TuistDependenciesTests/SwiftPackageManager/Utils/PackageInfoMapperTests.swift

```diff
@@ -1292,7 +1294,45 @@ final class PackageInfoMapperTests: TuistUnitTestCase {
             .test(
                 name: "Package",
                 targets: [
-                    .test("Target1", dependencies: [.xcframework(path: "/Path/To/Dependency1.framework")]),
+                    .test("Target1", dependencies: [.xcframework(path: "/artifacts/Package/Dependency1.xcframework")]),
+                ]
+            )
+        )
+    }
+
+    func testMap_whenBinaryTargetPathByNameDependency_mapsToXcFramework() throws {
```

Comment:
We should also add one for `whenBinaryTargetPathDependency` (i.e. standard SPM target dependency instead of ByName)

Reply:
feel free to add...